buildkite/scripts/build-artifact.sh

```diff
@@ -9,48 +9,26 @@ source ~/.profile
 
 MINA_COMMIT_SHA1=$(git rev-parse HEAD)
```

Comment:
Here i used makefile instead of repeating import of all app. Also reexported dune_instrument_with var to be sure it's defined if necesary